src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 16
+#define SOF_ABI_MINOR 17
```

Comment:
ABI - do we have kernel PR ?

Reply:
https://github.com/thesofproject/linux/pull/2087

Reply:
Will this PR work at MINOR 16 ? i.e. is there anything in 16 that this breaks ?

Reply:
This PR changes already used structure definition, and changes sof_ipc_window struct size calculation from kernel size - https://github.com/thesofproject/linux/pull/2087#discussion_r422852863 - so I bumped ABI. Even so, it's backward compatible.